### PR TITLE
fix: update image to support usage info

### PR DESCRIPTION
**Pull request by jeffmaury**
### What does this PR do?

Update the inference server to return usage statistics in streaming mode

### Screenshot / video of UI

N/A

### What issues does this PR fix or reference?

https://github.com/containers/podman-desktop-extension-ai-lab/issues/1730

### How to test this PR?

1. Build the image
2. Update the inference server.json file
3. Start the inference server for a model
4. Execute a curl query with streaming mode activated and usage info requested

### CODE REVIEW

**Comment by axel7083** [comment]
Referencing https://github.com/abetlen/llama-cpp-python/pull/1552 since we are using it

**Review by axel7083 — APPROVED** [review]
I honestly emit serious doubt to the longevity of such images. I don't think we should in the future build too much on the llama-cpp-python project.

File: chat/setup.sh
Comment (benoitf) [inline chat/setup.sh:5]: this PR is not merged
I think that you should bring the patch here as the PR may change one day to another so you'll never be able to see what will be used.

Or it means we wait until the PR is merged

Comment (jeffmaury) [reply chat/setup.sh]: Using rebase on the last commit of the PR instead of the PR diff for better stability

**Pull request merged**
